src/lib.rs

```diff
@@ -6,12 +11,20 @@ use std::fmt;
 use std::fs;
 use std::io::prelude::*;
 use std::ops::Deref;
+#[cfg(unix)]
 use std::os::unix::fs::MetadataExt;
+#[cfg(windows)]
+use std::os::windows::fs::MetadataExt;
 use std::path::{Path, PathBuf};
 use std::time;
 
 use mailparse::*;
 
+#[cfg(unix)]
+const INFORMATIONAL_SUFFIX_SEPARATOR: &str = ":";
+#[cfg(windows)]
+const INFORMATIONAL_SUFFIX_SEPARATOR: &str = ";";
```

Comment:
This is the biggest problem I encountered. Windows environments do not support the colon inside file names. I found nothing on the official doc, but I found this on [Wikipedia](https://en.wikipedia.org/wiki/Maildir#File-system_compatibility_issues):

> The Maildir standard can only be implemented on systems that accept colons in filenames.
>
> Systems that don't allow colons in filenames (this includes [Microsoft Windows](https://en.wikipedia.org/wiki/Microsoft_Windows) and some configurations of [Novell Storage Services](https://en.wikipedia.org/wiki/Novell_Storage_Services)) can use an alternative separator, such as ";" or "-". It is often trivial to patch [free and open-source software](https://en.wikipedia.org/wiki/Free_and_open-source_software) to use a different separator.

So I just tried a POC using a different separator and it seems to work. Now, the question is how to make this customizable? I thought about adding an entry in the `Maildir` struct (sth like `informational_suffix_separator`, or sth shorter like `info_suffix_delim`) with a default value set with the colon ":".

Let me know your opinion about this issue.

Reply:
Hm, interesting. I hadn't considered this problem before. Using the semicolon is fine, and I'm ok with not making this customizable right now. It doesn't look like there's a standard around what windows maildir software uses so for now there's no clear "best option" but we can make it customizable later if there's a need for it.

Reply:
Alright, so we let this hunk as it is right now?

Reply:
Yes please!